### ISSUE TITLE
Tab list broken on dev build 617

### ISSUE THREAD

**Issue by mibby** [CodeCrafter47/BungeeTabListPlus#160]
@CodeCrafter47 BungeeTabListPlus dev 617 breaks tab list on MC/Bungee 1.11. Downgrading back to dev 616, everything is fine.

Dev 617
![link](http://i.imgur.com/tQy7XL6.png)
Dev 616
![l2](http://i.imgur.com/89yAMk0.png)

```
customPlaceholders:
  # Defines the ${afk_tag} placeholder which is used to add "|away"
  # to the players name if he is afk.
  afk_tag:
    !conditional
    condition: ${player essentials_afk}
    true: '&7|&oaway'
    false: ''
  # Defines the ${viewer_colored_ping0} placeholder which displays the ping in green
  # if it is below 50ms, otherwise in yellow.
  viewer_colored_ping0:
    !conditional
    condition: "${viewer ping} < 100"
    true: "&a${viewer ping}"
    false: "&e${viewer ping}"
  # Defines ${the viewer_colored_ping} placeholder which displays the ping in red
  # if it is above 150ms, otherwise it is replaced with the ${viewer_colored_ping0}
  # placeholder ( < 50 -> green, > 50 -> yellow).
  # That results in the following color scheme:
  # 0 - 49   -> green
  # 50 - 149 -> yellow
  # 150+     -> red
  viewer_colored_ping:
    !conditional
    condition: "${viewer ping} < 200"
    true: ${viewer_colored_ping0}
    false: "&c${viewer ping}"

# Player sets are required to display players and player counts on the tab list.
# To display a player count use ${playerset:<name> size}, example: ${playerset:global size}.
playerSets:
  # The global player set contains all players
  global:
    filter: "true"
# # Some more examples. Be careful to get the number of spaces right when using them.
# # Player set containing all players on a specific server:
# survival:
#   filter: |-
#     ${player server} == "survival"
# # Player set containing all players on the same server as the player viewing the tab list:
# currentserver:
#   filter: |-
#     ${player server} == ${viewer server}
# # Player set containing all players on a specific server and world:
# survival_nether:
#   filter: |-
#     ${player server} == "survival"
#     and ${player world} == "world_nether"
# # Player set containing all admins:
# admins:
#   filter: |-
#     ${player vault_primary_group} == "admin"
#     or ${player vault_primary_group} == "owner"

# We want the tab list to have a fixed size of 60 slots.
# If you want a dynamic size you need to set type to DYNAMIC_SIZE
# and you don't need any of the options below it. Have a look at
# the example in the wiki to see which options you need instead:
# https://github.com/CodeCrafter47/BungeeTabListPlus/wiki/Examples#global-tab-list-dynamic-size
type: FIXED_SIZE
size: 60 # 60 slots -> 3 columns, 20 rows

# The defaultIcon and defaultPing will be used for all slots for which no other value is
# explicitely set.
defaultIcon: colors/dark_gray.png
defaultPing: 1000

# Here the content of the tab list is configured:
components:
# The players by server component adds players to the tab list grouped by server
- !players_by_server
  # The global player set is configured above
  playerSet: global
  playerOrder: "vaultprefix,alphabetically"
  # The server header is shown for each server above the players on that server.
  # We use it to display the server name and the player count.
  serverHeader:
  - {text: "&e&n${server}&f&o (${server_player_count}):", icon: "colors/yellow.png", ping: 0}
  # The server separator is shown between two consecutive servers. 
  # Here it use used to separate the servers with an empty row of slots, showing two more variants to display an empty slot.
  serverSeparator:
  - {}
  - ""
  -
  # Whether empty servers should show up too
  includeEmptyServers: true
  # Format of the player slot. Add prefixes as you like
  playerComponent: "${player vault_prefix}${player display_name}${afk_tag}"
  # If there isn't enough space for all players the morePlayersComponent is displayed. We use to to display the number of players which couldn't be displayed on the tab list.
  morePlayersComponent: {text: "&7... and &e${other_count} &7others", icon: "colors/gray.png", ping: 0}
# A spacer creates as many empty slots as possible.
- !spacer {}
# Because the spacer has eaten up all the remaining free slots, the nine slots below are guaranteed to be at the bottom of the tab list.
# Let's start with another empty row
#- "" # 18th row, 1st column
#- "" # 18th row, 2nd column
#- "" # 18th row, 3rd column
# Then some nice animations at the bottom
#- "" # 19th row, 1st column
#- "" # 19th row, 2nd column
#- "" # 19th row, 3rd column
- {text: "&fServer: &b${viewer server}", icon: "default/server.png", ping: 0} # 20th row, 1st column
- {text: "&fPlayers: &b${playerset:global size}", icon: "default/players.png", ping: 0} # 20th row, 2nd column
- {text: "&fPing: ${viewer_colored_ping}ms", icon: "default/ping.png", ping: 0} # 20th row, 3rd column
```

**Comment by CodeCrafter47** [comment]
I can't reproduce that issue.

**Comment by CodeCrafter47** [comment]
Turns out I was testing with the wrong version. Fixed by 50c4ab7bbd440dd71491f0fd2017550ead861f6e
